### DISCUSSION TITLE
deepCoadd-results detection footprints

### DISCUSSION THREAD

**Discussion by grantmerz** [LSSTDESC/desc-data-portal#97] [General]
Hello,

I have the deepCoadd-results data downloaded and would like to look at object footprints for my analysis. Is there a tutorial somewhere for accessing the information from the "det" FITS files?  From what I can tell, it seems I'll need to use LSST Science Pipelines software.  

Thanks!

**Comment by heather999** [comment]
HI @grantmerz Sorry for the long delayed response.  We've been reminded that the data we have available here publicly is an older format than what is being produced now via modern versions of the LSST Science Pipelines.  

As a DESC member, you have the ability to become a Rubin DP0 delegate where Rubin has actually made DESC's DC2 data available. There are spots still open for DP0 delegates and once set up, you can access the Rubin Science Platform (RSP) to run a set of tutorial notebooks on what they call their "Notebook Aspect" which is basically Jupyter. That's probably a better use of your time so you're using the more recent versions of the LSST Science Pipelines and becoming familiar with the interfaces that Rubin will be providing.  And you can still work with DESC DC2 data, as well as additional data that Rubin will be making available.
Rubin has provided a link a site that includes a checklist that you can follow to get set up with the RSP:
https://dp0-2.lsst.io/
See specifically:
New to DP0? Welcome! See the [Getting started with DP0 checklis](https://dp0-2.lsst.io/dp0-delegate-resources/index.html#delegate-homepage-getting-started-checklist)

To help answer your more specific question about accessing the object footprints.. there is a GitHub repo managed by a group called "StackClub" which has put together a nice set of example notebooks: https://github.com/LSSTScienceCollaborations/StackClub/blob/master/SourceDetection/Footprints.ipynb

Just let me know if you need any help or have any questions!

**Reply by grantmerz** [reply]
Thank you for the response!  I became a DP0 delegate, getting familiar with the data/software and I've been able to access the footprints on the RSP.  I would ultimately like to get a data set I can use for some ML training, but I am not sure if that's possible with DP0 data as I'll need to download the data onto a local machine for GPU jobs.  Ultimately I'd like to make a public repo with results/data, which is why I started with the DC2 public data release, but either way I'll need to move some data around.  I posted about this [on the community forum](https://community.lsst.org/t/deblended-footprints-for-dc2-data/7735?u=grantmerz) as well.
